the-events-calendar/event-tickets
# Merge bucket/ft-fast-follow in release branch

- build(Common) update to bucket branch
- feat - remove helper text and link from main template
- feat - removed helper text for series passs
- tests - integration snapshots updated
- tests - ft_integration snapshots updated
- feat(Repository) add ticket type filter to Attendees, complete Tickets
- build(Common) track latest
- fix(Ticket_Repository) merge issue
- feat - removed helper text methods
- feat - updated warning message text with template
- feat - updated warning template
- feat - updated css
- feat - FT message filter
- feat - FT warning messages updated
- Revert "feat - FT message filter"
- feat - add series pass filters
- common updated
- tests - ft_integration snapshots updated
- track common
- css updated
- tests - integration snapshots updated
- phpcs
- tests - snapshots updated
- feat - extra class removed
- feat - hide series pass regular info on recurring event
- tests - snapshots updated
- tests - snapshots updated
- tests - fix restv1
- tests - fix ETP Woo sharedcaptest fails
- tests - shared cap tests expectations updated
- tests - fix expectation
- feat - update template tags
- feat - updated attendees in order report
- tests - integration snapshots updated
- feat - use plural tickets
- tests - updated snapshots
- feat - updated block instances
- tests - integration updated
- update to RSVPs
- track common
- Revert "track common"
- snapshots updated
- feat - update metabox editor
- tests - snapshots updated
- fix(Metabox) use correct case for class name to count active providers
- test(ft_integration) cover no providers case in FT context
- style(phpcbf) fix
- tweak - apply cr changes
- tweak - allow `rel` attribute
- tests - integration snapshots updated
- tests - ft_integration snapshots updated
- tests - snapshots updated
- tests - wpunit snapshots
- feat(FT) per Event pass checkin support
- style(Series_Passes) add missing blank line
- feat - type description
- tests - snapshots updated
- WiP - clone Series Pass Attendee on checkin
- build(Common) track latest
- build(Common) track latest feature branch
- feat - update link vars
- feat - updated template
- tests - snapshots updated
- track common
- feat - link updated
- tests - snapshots updated
- feat - update warning message with filter
- tests - updated warning notice related tests
- feat - filter warning message from series pass
- tests - snapshots updated
- tests - fixed label and snapshots updated
- tests - ft_integration updated
- build(Common) track latest
- feat - updated is_event_page logic
- feat - removed optional param from calls
- fix typo
- doc blocks updated
- track common
- feat - use filter to include rewrites
- apply cr changes
- removed optional param
- build(Common) track latest
- feat(FT) clone Attendee on Pass checkin, handle clone updates
- tests added
- snapshots updated
- placehold
- build(Common) track bucket/ft-fast-follow-branch of Common
- fix(FT) ensure Attendee > Event relationship key updated on provisional ID update
- :fast_forward: https://github.com/the-events-calendar/tribe-common/commit/cc2275d3badcb44fa85da94cabf86bed3ff1f02b
- refactor(FT) move Attendee functionality to dedicated controller
- feat(FT) filter event-based queries for Attendees to handle clones
- build(Common) track latest from feature branch
- :fast_forward: https://github.com/the-events-calendar/tribe-common/commit/c3275512e3b170d1a5cd6a8a1ce11daccb17a4b8
- style(src) fix a couple files
- updated links
- build(.env.testing.slic) add file
- tests - snapshots updated
- refactor(RSVP) remove func_get_args argument parsing
- :fast_forward: https://github.com/the-events-calendar/tribe-common/commit/2883005ffadacad3f7b2c85ce0060d5586881093
- build(Common) track latest
- build(.github) remove old files and dirs
- fix(FT) ensure event ID is passed down correctly to check-in; refactor to unify use of filtering function
- fix(FT) simplify event ID filtering logic
- fix(FT) reload Attendees list on clone
- fix(Attendance Total) recalculate totals when a new Event ID is set
- test(ft_integration) update snapshots following correct recalculation
- style(FT) add missink line break
- fix(FT) handle check-in request from Series context correctly
- style(src,tests) apply phpcbf fixes
- fix(FT) ensure Att. ref. Single Events by post ID
- style(src,tests) fix code style issues
- updated tickets to attendees
- snapshots updated
- phpcbf fixes
- snapshots updated
- phpcs fix
- test(ft_integration) fix ticket order to stabilize snapshot
- remove checkin row action
- test(ft_integration) cover checkin row action removal
- style(PHPCS) fixes
- :fast_forward: https://github.com/the-events-calendar/tribe-common/commit/ebbd56d85debef1e2c08e4d82e30b2689ba5e607
- build(Common) track latest
- Set version to 5.8.2
- doc(reamde.txt) add FT fast-follow work changelogs
- :fast_forward: https://github.com/the-events-calendar/tribe-common/commit/ac520f503c6299668109a7f8a0d8bd25b1b562da
